x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/create_package_config_page/step_define_package_config.tsx

```diff
@@ -47,9 +47,16 @@ export const StepDefinePackageConfig: React.FunctionComponent<{
         .sort();
 
       updatePackageConfig({
-        name: `${packageInfo.name}-${
-          dsWithMatchingNames.length ? dsWithMatchingNames[dsWithMatchingNames.length - 1] + 1 : 1
-        }`,
+        name:
+          // For Endpoint packages, the user must fill in the name, thus we don't attempt to generate
```

Comment:
Can you add a FIXME comment to improve this in the next release? :) I'm not sure if there is an issue filed but adding the URL of the issue would be great too.